client/my-sites/site-settings/site-owner-transfer/start-site-owner-transfer.tsx

```diff
@@ -27,18 +33,185 @@ const SiteOwnerTransferActionPanelBody = styled( ActionPanelBody )( {
 	overflow: 'visible !important',
 } );
 
+const Title = styled.h2( {
+	fontWeight: 'bold',
```

Comment:
```suggestion
	fontWeight: '500',
```

Let's set this to the same font-weight as in Figma file